SU-HSDO/stanford_mrc
# MRCD8-159 Set display settings for event nodes

# READY FOR REVIEW

# Summary
- Event node display settings

# Needed By (Date)
- Today

# Urgency
- High

# Steps to Test

1. checkout branch.
2. drush updb
3. create an event
4. verify it looks kinda like https://projects.invisionapp.com/share/DCEGPUZS4#/screens/263924799

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
